astropy/wcs/setup_package.py

```diff
@@ -9,7 +9,7 @@
 from os.path import join
 from pathlib import Path
 
-import numpy
+import numpy as np
```

Comment:
Doesn't really improve clarity here....

Reply:
I don't see what's unclear about `import numpy as np`.

Reply:
_"Doesn't really improve clarity here"_ does not imply (at least to me) that `import numpy as np` is "unclear". But in this particular case, I do find `as np` an unnecessary alias.

Reply:
The point is, I think, that even if it is not (much) unclearer, there was a conscious decision to choose this style even when the `np` alias was already standard usage, possibly to emphasise the correspondence to `include_dirs=["numpy"]` (#9726).

Reply:
> Agreed with @dhomeier - I prefer not to be distracted by unnecessary `noqa`. So, we
> 
> 1. Work around this rule with `from numpy import get_include as get_numpy_include`;
> 2. Ignore `setup_package.py`; or
> 3. Just make the few good changes here and simply keep ignoring `ICN001` - this is _not_ a big deal one, and new contributions are very unlikely to "break" it.


We have changed this to `from numpy import get_include as get_numpy_include` since the package is not being used itself, only its include files are needed.